test/greet-test.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const greet = require('../lib/greet.js');
+const assert = require('assert');
+var name = process.argv[2];
+
+describe('Greet Module', function() {
+  describe('#sayHello', function() {
+    it('should return hello {name}!', function() {
+      var result = greet.sayHello(name);
+      assert.ok(result === `hello ${name}!`, 'not equal to hello peter!');
+    });
+    it('should throw a missing name error', function() {
+      assert.throws(function() {
+        greet.sayHello();
+      }, 'error not thrown');
+    });
+  });
+});
```

Comment:
You may have discovered that mocha does not accept command line arguments, thus you can't technically test process.argv[2] in the way you would expect. This works, however, it would better to check for a truthy value since process.argv[2] in mocha will always be undefined! Don't worry about this though, we will get into that later this week :)